experimental/traffic-portal/src/app/core/users/users.component.ts

```diff
@@ -229,8 +229,8 @@ export class UsersComponent implements OnInit {
 			newTab: true
 		},
 		{
-			action: "viewChangelogs",
-			disabled: (): true =>true,
+			disabled: (us: GetResponseUser | Array<GetResponseUser>): boolean => Array.isArray(us),
```

Comment:
if `multiRow` isn't `true`, the action is implicitly disabled when multiple rows are selected, so you shouldn't need to define this. If you do, that's a bug in the generic table component.